src/main/java/com/xjeffrose/chicago/ChiUtil.java

```diff
@@ -13,6 +13,8 @@
   private static final Logger log = LoggerFactory.getLogger(ChiUtil.class.getName());
   public static final String delimiter = "@@@";
   public static final String defaultColFam = "chicago";
+  public static final String TS = "timeSeries";
+  public static final String KV = "KV";
```

Comment:
The data that is written along with the path in ZK. Lets me know whether it is TimeSeries, in which case I could just stream from the beginning else I can scan the column family for the candidate keys.